### ISSUE TITLE
Create research_summary entity

### ISSUE THREAD

**Issue by roll** [opentrials/opentrials#242]
HRA data now goes to publication but it's probably should be moved to `research_summary` entity.

**Comment by roll** [comment]
@opentrials/research 
So here is a question to domain specialists:

As a {User} I want to see data about research summary with following attributes:
- name
- authors
- ???

We need this data to model new entity.

**Comment by roll** [comment]
also related to https://github.com/opentrials/opentrials/issues/244

**Comment by vitorbaptista** [comment]
@opentrials/research How should the research summaries be displayed on a trial page? Currently, they're displayed the same way as a publication. Maybe we could simply add some indication that the publication is a research summary, which is a pattern we can follow for systematic reviews as well. For example:

![publications separated by categories](https://cloud.githubusercontent.com/assets/76945/17184116/82541704-5422-11e6-9366-e93077575253.png)

With some better separation to make it clear what's a publication and what's a category name (research summary, systematic review, etc.), or even just an icon or small badge near the title to show what it is.

Thoughts?

**Comment by benmeg** [comment]
Would vote for the [PDF icon](https://wwwimages2.adobe.com/content/dam/acom/en/legal/images/badges/Adobe_PDF_file_icon_32x32.png) to denote Papers and something like the [Cochrane logo](http://www.cochrane.org/sites/default/files/icons/cochrane-icon.png) for Systematic Reviews. No immediate suggestions for Research Summaries.

**Comment by roll** [comment]
If we will be able to have one entity `publication` for all this stuff it will simplify things.

**Comment by Bengoldacre** [comment]
We get two important things from the HRA data that is matched on.

1. a lay summary (possible same, possibly different, could maybe invite people to up/downvote them in comparison to other lay summaries from eg registry)

2. the HRA application process paperwork, under the heading "trial approval documents".

These both need to be on page, separately.

**Comment by vitorbaptista** [comment]
@Bengoldacre So your suggestion is to add all lay summaries on the "main" trial page (e.g. http://explorer.opentrials.net/trials/b02b35ca-68a1-49e0-8848-4eb3c142e503), in our case both the one from the registry and the one from HRA?

Also, where can I find the "HRA application process paperwork"? Couldn't find it on HRA's research summary page (e.g. http://www.hra.nhs.uk/news/research-summaries/24-hour-labour-line-a-qualitative-exploration/)

**Comment by vitorbaptista** [comment]
@Bengoldacre @opentrials/research Please, when you have some time, answer my question at https://github.com/opentrials/opentrials/issues/242#issuecomment-239363393. This issue is currently blocked waiting for more information.

**Comment by benmeg** [comment]
> @Bengoldacre So your suggestion is to add all lay summaries on the "main" trial page (e.g. http://explorer.opentrials.net/trials/b02b35ca-68a1-49e0-8848-4eb3c142e503), in our case both the one from the registry and the one from HRA?

IIRC, we said we'd display one (wasn't too important which one for now), but not both, to avoid overwhelming the user. Personally I would vote for a tabbed UI for trials with >1 lay summaries (maybe with some +/- voting buttons on each tab, which after X number of votes determines which summary is shown in the default tab?).

> Also, where can I find the "HRA application process paperwork"? Couldn't find it on HRA's research summary page (e.g. http://www.hra.nhs.uk/news/research-summaries/24-hour-labour-line-a-qualitative-exploration/)

I couldn't see any links to application process paperwork either, but @Bengoldacre mentioned the issue in this email thread (but the example seem to be missing?):

https://groups.google.com/a/okfn.org/d/msg/opentrials-team/Wf8VV1g8bYc/aBx1ueKSBwAJ

However, he *does* mention other metadata: "rec number, IRAS number, approval date, etc."

Any other thoughts @opentrials/research?

**Comment by jessflem** [comment]
Yep yep, I agree with the above. We don't have access to the paperwork
through the site, so ignore the "HRA application process paperwork" for
now. We do want all the other metadata on the hra summary pages to be
collected, rather than just the summary.

Thanks!

**Comment by vitorbaptista** [comment]
Just talked with @jessflem about this. Our plan is to create the `research_summary` entity with all the fields from the HRA page (e.g. http://www.hra.nhs.uk/news/research-summaries/longterm-fu-study-of-botox-in-idiopathic-overactive-bladder-patients/). I'm concerned that it'll tie us too much to the HRA research summaries, but we can abstract the details later when we get other sources of research summaries. We'll then create a page similar to what they have, but inside OpenTrials.

The link in the trials page will be similar to what we have for publications, but named "Research summaries", and linking using the "HRA" name, as:

![research summaries](https://cloud.githubusercontent.com/assets/76945/18912473/790d082a-857a-11e6-800f-824d03f13646.png)

For the launch, we agreed on having just these links pointing to the page in the HRA website (https://github.com/opentrials/opentrials/issues/415).

**Comment by pwalsh** [comment]
@vitorbaptista still relevant?
